models/multimeter.cpp

```diff
@@ -176,8 +215,8 @@ Multimeter::update( Time const& origin, const long from, const long )
   // following Reply data is then added.
   //
   // Note that not all nodes receiving the request will necessarily answer.
-  V_.new_request_ =
-    B_.has_targets_ && !P_.record_from_.empty(); // no targets, no request
+  V_.new_request_ = B_.has_targets_
+    && P_.record_from_.empty() == false; // no targets, no request
```

Comment:
One should not compare with `true` and `false`. The following would be better (comment moved for better line structure):
```c++
// no targets or no recordables means no request
V_.new_request_ = B_.has_targets_ and not P_.record_from_.empty();
```